components/script/dom/node.rs

```diff
@@ -349,7 +354,16 @@ impl Node {
             kid.teardown();
         }
     }
+    /// method on Node that returns Vec<Root<MutationObserver>>
+    pub fn registered_mutation_observers_for_type(&self) -> &DOMRefCell<Vec<MutationObserver>> {
+         return &self.mutation_observers ;
+    }
 
+    /// add a registered observer
+    pub fn add_registered_mutation_observer(&self, observer: MutationObserver) -> Fallible<()> {
```

Comment:
This should be a `&MutationObserver` argument, and it does not need a return value.